### PR TITLE
#933 Add Strava API

### PR DESCRIPTION
**Pull request by pnowy**
Added Strava API

### CODE REVIEW

File: scribejava-apis/src/test/java/com/github/scribejava/apis/examples/StravaExample.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        System.out.println("And paste the authorization code here");
+        System.out.print(">>");
+        final String code = in.nextLine();
+        System.out.println();
+
+        System.out.println("Trading the Authorization Code for an Access Token...");
+        final OAuth2AccessToken accessToken = service.getAccessToken(code);
+        System.out.println("Got the Access Token!");
+        System.out.println("(The raw response looks like this: " + accessToken.getRawResponse() + "')");
+        System.out.println();
```
Comment (kullfar) [inline scribejava-apis/src/test/java/com/github/scribejava/apis/examples/StravaExample.java]: it's better to show any API call with this access_token

**Comment by kullfar** [comment]
Hi,
Thanks for the Pull Request.
I need test app credentials and test user credentials to run the example.
https://github.com/scribejava/scribejava/wiki/FAQ#when-will-you-support-insert_api_here-

Thanks in advance

File: scribejava-apis/src/test/java/com/github/scribejava/apis/examples/StravaExample.java (hunk 1 above)
Comment (pnowy) [reply scribejava-apis/src/test/java/com/github/scribejava/apis/examples/StravaExample.java]: Hi,

I've added example API call with access token.

**Comment by pnowy** [comment]
You should receive the example credentials on your email.

**Comment by pnowy** [comment]
@kullfar any update on this? Did you receive email from me?

**Comment by kullfar** [comment]
Hi, sorry for the delay. I got the email from you with app test credentials.
I need test user credentials as well

**Comment by kullfar** [comment]
Closing as out-dated. Feel free to reopen (and provide all test credentials, please. thx in advance).

**Pull request closed without merging**